gulpfile.js

```diff
@@ -24,9 +24,7 @@ function onError(error) {
 }
 
 let projectTypings = ts.createProject('src/tsconfig.json');
-let projectCommonjs = ts.createProject('src/tsconfig.json', {
-    target: 'es6',
-});
```

Comment:
I already checked that the mentioned `tsconfig.json` already has `target:es6` so you don't have to worry about me deleting this line :rose: